File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2949.c
#include <stdio.h>

int main(){
    int n, i, a=0, elf=0, h=0, m=0, x=0;
    char nome[128], e;
    scanf("%d", &n);
    for(i=0;i<n;i++){
        scanf("%s %c", nome, &e);
        if(e=='A') a++;
        else if(e=='E') elf++;
        else if(e=='H') h++;
        else if(e=='M') m++;
        else x++;
    }
    printf("%d Hobbit(s)\n", x);
    printf("%d Humano(s)\n", h);
    printf("%d Elfo(s)\n", elf);
    printf("%d Anao(s)\n", a);
    printf("%d Mago(s)\n", m);

    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2949.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner ler = new Scanner(System.in);
        int n, i, a=0, elf=0, h=0, m=0, x=0;
        String nome, e;
        n = ler.nextInt();
        for(i=0;i<n;i++){
            nome = ler.next();
            e = ler.next();
            if(e.equals("A")) a++;
            else if(e.equals("E")) elf++;
            else if(e.equals("H")) h++;
            else if(e.equals("M")) m++;
            else x++;
        }
        System.out.printf("%d Hobbit(s)\n", x);
        System.out.printf("%d Humano(s)\n", h);
        System.out.printf("%d Elfo(s)\n", elf);
        System.out.printf("%d Anao(s)\n", a);
        System.out.printf("%d Mago(s)\n", m);

    }
}